agent/app/agent_test.go

```diff
@@ -78,6 +79,20 @@ var apiVersions = []dockerclient.DockerVersion{
 	dockerclient.Version_1_22,
 	dockerclient.Version_1_23}
 var capabilities []*ecs.Attribute
+var testHostCPU = int64(1024)
```

Comment:
nit. Should they be constants instead of variables since we are not going to change them for different test cases?

Reply:
if we make those constants, then testHostCPU and testHostMEMORY are not addressable, we can not get address of them in `var testHostResource = map[string]*ecs.Resource{ .....IntegerValue: &testHostCPU,}`